spring-cloud-gcp-docs/src/main/asciidoc/datastore.adoc

```diff
@@ -836,6 +836,27 @@ You can also query for non-entity types:
 	String getOneId(@Param("id_val") long id);
 ----
 
+Kind names can also be resolved from the `@Entity` annotation on domain classes.
```

Comment:
What does the "also" refer to? I don't see any discussion about other ways to get kind names right before this paragraph.
In any case, how is this `|com.example.Trade|` syntax relevant to SpEL? I don't remember seeing anything like this in other Spring Data implementations.

Reply:
Also implies there are at least 2 methods.

methods 1 is directly writing the name in the string.
method 2 is using the java class name. 

This is a feature from spring data JPA.

The reason SpEL is here is because the Kind name annotation for the class can have SpEL expressions  in it.